sklearn/linear_model/passive_aggressive.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, C=1.0, fit_intercept=True,
                                    n_jobs=n_jobs)
         self.C = C
         self.loss = loss
+        self.alpha = 1.0
```

Comment:
Not yet, thanks for tagging it again though. I have some time this weekend so I'm going to work on it.